server/container_create_linux.go

```diff
@@ -906,6 +913,95 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrfactory.Cont
 	return ociContainer, nil
 }
 
+func setContainerTimeZone(ctx context.Context, tz, mountPoint, etc, mountLabel string, options []string, ociContainer *oci.Container, ctr ctrfactory.Container) error {
+	var err error
+	if tz == "" {
+		return nil
+	}
+
+	timezonePath := filepath.Join("/usr/share/zoneinfo", tz)
```

Comment:
We should either make this configurable or parse the `TZDIR` environment variable: https://sourceware.org/git/?p=glibc.git;a=blob;f=time/tzfile.c;h=8a923d0cccc927a106dc3e3c641be310893bab4e;hb=HEAD#l149